src/components/charts/Charts.tsx

```diff
@@ -75,13 +76,15 @@ export default function Charts() {
     return longestWord;
   }
 
+
+  const isMobileDevice = useMediaQuery({ maxDeviceWidth: 1200 })

```

Comment:
Would it be better to assess what type of device we're working with once on page load rather than on each component render like this?